common.py

```diff
@@ -51,14 +51,24 @@ def get_branches(repo):
         print("No branches of interest was defined, using all branches with "
               "names starting with a number or v[0-9] ")
 
-        from github import Github
-        from common import get_credentials
-        g = Github(*get_credentials())
-        repo = g.get_repo(repo)
+        try:
+            from github3 import GitHub, AuthenticationFailed
+        except ImportError:
+            raise ImportError('Please conda or pip install github3.py')
+        # from common import get_credentials  # Not needed?
```

Comment:
Note: I don't know if there is an special use case that need this but I don't see why it needs to import a function that is implemented in the same file here.